docker/dkeyserver/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin sh
+sh /home/sgx_config.sh
+sh -c /home/ehsm/out/ehsm-dkeyserver/ehsm-dkeyserver
```

Comment:
combine and named as start_dkeyserver.sh

Reply:
It will be fixed